cura/MachineAction.py

```diff
@@ -49,17 +59,24 @@ def setLabel(self, label: str) -> None:
             self._label = label
             self.labelChanged.emit()
 
-    ##  Reset the action to it's default state.
-    #   This should not be re-implemented by child classes, instead re-implement _reset.
-    #   /sa _reset
     @pyqtSlot()
     def reset(self) -> None:
+        """Reset the action to it's default state.
+        
+        This should not be re-implemented by child classes, instead re-implement _reset.
+
+        :py:meth:`cura.MachineAction.MachineAction._reset`
```

Comment:
Weirdly py:class works, but this gives:
```
System Message: ERROR/3 (<stdin>, line 19); backlink

Unknown interpreted text role "py:meth".
```

In the pycharm rst file preview. In the ctrl+q code quick doc it renders as regular text so it might just be acceptable.Is it correct, is some extra rst module required?

Reply:
Also, maybe a "See also:" prefix would be nice to indicate why you put it there

Reply:
`:py:meth:` is correct rst see [cross-referencing-python-objects](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-python-objects) but I think pycharm hasn't implemented those yet, although it has been requested 6 years ago [PY-13902](https://youtrack.jetbrains.com/issue/PY-13902)

Reply:
I will as the `See also`